Makefile

```diff
@@ -71,11 +104,13 @@ python: mpi4py
 endif  # ENABLE_MPI
 
 
-$(VENV):
+# Due to dependency issues with Python3.12 and Muscle3, we create a VENV with the highest available python less than 3.12
```

Comment:
I think this was fixed in MUSCLE3 0.7.2, but it seems like you reverted back to 0.7.1? Is there anything I can do to help with that? Perhaps the problem was that it was updated on the Python side but not the C++ side and the versions ended up incompatible?

Reply:
Indeed the issue was fixed in MUSCLE3, but building still failed later on with the ymmsl package with a similar error caused by deprecated functions in the new numpy version.